src/onINP.ts

```diff
@@ -93,11 +86,13 @@ export const onINP = (
       // running in Chrome (EventTimingKeypressAndCompositionInteractionId)
       // 123+ that if rolled out fully may make this no longer necessary.
       whenIdle(() => {
-        entries.forEach(processInteractionEntry);
+        for (const entry of entries) {
+          processInteractionEntry(entry);
+        }
 
         const inp = estimateP98LongestInteraction();
 
-        if (inp && inp.latency !== metric.value) {
+        if (inp?.latency !== metric.value) {
```

Comment:
seems a little risky but I guess `metric.value` should never be undefined...

Reply:
Yeah, `metric.value` should also be a number ... but looking at this again, you're right that it is subtly different and maybe it's better not to use optional chaining in these cases? WDYT?

Reply:
Decided to just revert this since the original code minifies better anyway...